mpas_xarray.py

```diff
@@ -91,16 +119,40 @@ def test_load_mpas_xarray_datasets(path):
 
     return
 
+def test_load_mpas_xarray_timeSeriesStats_datasets(path):
+    ds = xr.open_mfdataset(path, preprocess=preprocess_mpas_timeSeriesStats)
+    ds = remove_repeated_time_index(ds)
+    ds2 = xr.open_mfdataset(path, preprocess=preprocess_mpas)
+    ds2 = remove_repeated_time_index(ds2)
+
+    # make a simple plot from the data
+    var = ds["timeSeriesStatsMonthly_avg_iceAreaCell_1"]
+    var2 = ds2["timeSeriesStatsMonthly_avg_iceAreaCell_1"]
+    ind_ice  = var > 0
+    ind_ice2 = var2 > 0
+    var_ice  = var.where(ind_ice)
+    var_ice =  var_ice.mean('nCells')
+    var2_ice = var2.where(ind_ice2)
+    var2_ice = var2_ice.mean('nCells')
```

Comment:
You could write a simple inline function to tie together this redundant code if you wanted.  This could help with readability, e.g., something like

```python

def sel_data(ds):
     var = ds["timeSeriesStatsMonthly_avg_iceAreaCell_1"]
     indice  = var > 0
     varice  = var.where(indice)
     return varice

sel_data(ds).plot()
sel_data(ds2).plot()
...
```